scalableminds/webknossos
# Redesign "Welcome" toast

PR redesigns the "Welcome" toast for new, anonymous users to be in line with the new branding.

<img width="1627" alt="image" src="https://user-images.githubusercontent.com/1105056/229810052-005df3d9-ee6c-434e-a3f2-df485ff692da.png">


### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Set dev environment to Wkorg
- Share a dataset with yourself
- Open DS in incognito mode 
- Check out the redesigned welcome toast

### Issues:
- contributes to #6739 

------
(Please delete unneeded items, merge only when none are left open)
